scripts/download_test_resources.py

```diff
@@ -40,7 +40,7 @@ def __parse(self):
         p.add_argument(
             "--top-hash",
             # Generated package hash from upload_test_resources
-            default="54a874450af2b1cda2910420345145fbe49e0fe5347b944c2d294c4f3c892ee5",
+            default="8f6584820cbadc32d7eb606dfa70510c081e8442c91ce45af76336a6761c0504",
```

Comment:
* Adds the multi-scene pyramid CZI + OME TIFF files
* Removes the 6GB LIF file in hopes we can get a new "variable scene shape LIF" before I work on 4.x `LifReader`